src/containers/ProfilePage/ProfilePage.js

```diff
@@ -142,6 +151,50 @@ export const DesktopReviews = props => {
   );
 };
 
+export const CustomUserFields = props => {
+  const { publicData, metadata, userFields } = props;
+  return (
+    <>
+      <H4 as="h2" className={css.listingsTitle}>
+        <FormattedMessage id="ProfilePage.customFieldsHeading" />
+      </H4>
+      <SectionDetailsMaybe {...props} />
+      {userFields.reduce((pickedElements, config) => {
+        const { key, enumOptions, includeForListingTypes, scope = 'public' } = config;
+        // TODO fix isTargetListingType => isTargetUserType
+        const listingType = publicData?.listingType;
+        const isTargetListingType =
+          includeForListingTypes == null || includeForListingTypes.includes(listingType);
```

Comment:
At minimum, I think this should be handled (or commented out with exposed `const isTargetListingType = true;`)
However, check my larger comment on this setup.

Reply:
Addressed in moving this to helper!